README.md

```diff
@@ -62,6 +62,20 @@ you'll need to install both Ruby and the Ruby DevKit for compiling native extens
     gem install showoff
 
 
+Alternatively, you can run Showoff using docker:
+
+    docker run --rm -it -p 9090:9090 -v $(pwd):/var/cache/showoff binford2k/showoff
+
+
+Or build the image yourself:
+
+    docker build -t Dockerfile local/showoff .
```

Comment:
Hi, I just stumbled upon this project and thankfully over your Merge Request. 
I did try out the Docker build. It works like a charm.
There is an error though in this line, the flag "-f" is missing. This line is correct:

docker build -f Dockerfile -t local/showoff .

I would also suggest overwriting the Dockerfile under build/Dockerfile as you mentioned.

Reply:
Hi @lvoss Glad you found this helpful.  

As for the -f flag, I may be missing something, but I believe the default value of -f is "Dockerfile" and that -f is only needed if using a name other than "Dockerfile".  I'd be interested to see the error you're getting if you're not able to build without the -f flag _and_ the file is named "Dockerfile".  Are you using the latest version of the docker CLI?

And as for where to put this Dockerfile, I'm waiting to hear what the maintainer wants to do.  I agree replacing the build/Dockerfile may be best, since it doesn't appear to be working properly.

Reply:
Ah, now I see what the problem is.  I have the `-t` in the wrong place.  I'll fix it. Thanks for catching it.